<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NextGame</class>
 <widget class="QDialog" name="NextGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background-color:orange;
}</string>
  </property>
  <widget class="QLabel" name="quater">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>140</y>
     <width>551</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color:rgb(255,255,255);
background-color:rgb(0,0,0);
border: 3px solid white;
}</string>
   </property>
   <property name="text">
    <string>Quarter</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>660</y>
     <width>595</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="sub">
      <property name="text">
       <string>Substitutions</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="tactics">
      <property name="text">
       <string>Tactics</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="def_tactic">
      <property name="text">
       <string>Default Tactic</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>0</y>
     <width>551</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="away_name">
        <property name="styleSheet">
         <string notr="true">QLabel{
color:rgb(255,255,255);
background-color:rgb(0,0,0);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="away_score">
        <property name="styleSheet">
         <string notr="true">QLCDNumber{
color:rgb(0, 0, 255);
background-color: rgb(0, 0, 0);
border: 3px solid white;
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="time">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:yellow;
background-color:rgb(0,0,0);
border: 4px solid white;
border-radius: 7px;
}</string>
      </property>
      <property name="text">
       <string>00:00</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="home_name">
        <property name="styleSheet">
         <string notr="true">QLabel{
color:rgb(255,255,255);
background-color:rgb(0,0,0);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="home_score">
        <property name="styleSheet">
         <string notr="true">QLCDNumber{
color:rgb(255, 0, 0);
background-color: rgb(0, 0, 0);
border: 3px solid white;
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="end_msg">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>590</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="second_q">
   <property name="geometry">
    <rect>
     <x>957</x>
     <y>640</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Go to 2nd</string>
   </property>
  </widget>
  <widget class="QPushButton" name="start">
   <property name="geometry">
    <rect>
     <x>957</x>
     <y>610</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Start </string>
   </property>
  </widget>
  <widget class="QPushButton" name="third_q">
   <property name="geometry">
    <rect>
     <x>1077</x>
     <y>610</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Go to 3rd</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fourth_q">
   <property name="geometry">
    <rect>
     <x>1077</x>
     <y>640</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Go to 4th</string>
   </property>
  </widget>
  <widget class="QPushButton" name="end_game">
   <property name="geometry">
    <rect>
     <x>957</x>
     <y>670</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>End</string>
   </property>
  </widget>
  <widget class="QLabel" name="comments">
   <property name="geometry">
    <rect>
     <x>294</x>
     <y>285</y>
     <width>611</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="quit">
   <property name="geometry">
    <rect>
     <x>1077</x>
     <y>670</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="table_away">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>191</y>
     <width>231</width>
     <height>241</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>Overall</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Attack</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Defence</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Infos</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="table_home">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>191</y>
     <width>231</width>
     <height>241</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>2</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>3</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>4</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>5</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Players</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Overall</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Energy</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
